docs/releaseNotes.yml

```diff
@@ -43,6 +43,18 @@ items:
           release of 1.23. This provides $productName$ with the latest security patches, performances enhancments,
           and features offered by the envoy proxy.
         docs: https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.23/v1.23.0
+      - title: Fixed <code>mappingSelector</code> associating <code>Hosts</code> with <code>Mappings</code>
+        type: change
```

Comment:
I'm OK calling this a `bugfix`, not a `change`. (But at the same time, doing it in a .z is questionable)

Reply:
Yeah while it is a bugfix, I don't want someone to be skimming the release notes and go "oh bugfix, that's probably just an improvement" then not look into it too much when the change in behaviour may actually be really relevant to them.